docs/General/getting-started/getting-started-with-bpmn-js.md

```diff
@@ -10,3 +10,254 @@ grand_parent: "General"
 ---
 
 # Getting Started With bpmn-js
+
+bpmn-js is a [BPMN 2.0](https://www.omg.org/spec/BPMN/2.0/About-BPMN/) rendering toolkit and web modeler. It is written in JavaScript, embeds BPMN 2.0 diagrams into modern browsers and requires no server backend. That's why it is easy to embed it into any web application.
+
+Feel free to play with it [on our demo playground](https://demo.bpmn.io/s/start).
+
+![image](/assets/images/bpmn-start.gif)
+
+bpmn-js can be included in any application with:
+
+* an [all-in-one pre-packaged version of the library](https://unpkg.com/browse/bpmn-js/dist/)
+* the [npm](http://npmjs.com/package/bpmn-js) version, which gives you access to individual library components and allows for easier extensibility
+
+## Including the Root Element
```

Comment:
Don't call this root element. It's easy to confuse this with the diagram's root element. This one is simply the container/parent.

Reply:
Done 👍